KicadModTree/nodes/specialized/ExposedPad.py

```diff
@@ -105,11 +105,22 @@ class ExposedPad(Node):
           0 means no rounding
 
         * *radius_ratio* (``float``) --
-          The radius ratio for all pads (copper, paste and mask)
-          Default: 0 means pads do not included rounded corners (normal rectangles are used)
+          The radius ratio of the main pads.
         * *maximum_radius* (``float``) --
-          Only used if a radius ratio is given.
-          Limits the radius.
+          The maximum radius for the main pads.
+          If the radius produced by the radius_ratio parameter for the pad would
+          exceed the maximum radius, the ratio is reduced to limit the radius.
+        * *round_radius_exact* (``float``) --
+          Set an exact round radius for the main pads.
+
```

Comment:
Is this empty line intentional?

Reply:
yes
separates the description of the copper pad handling parameters from the paste handling ones

Reply:
There is no empty line in `Pad.py` before the paste stuff.

Reply:
There are now